### DISCUSSION TITLE
Nous A1Z does not show firmware release in OTA screen

### DISCUSSION THREAD

**Discussion by plevrier** [Koenkk/zigbee2mqtt#20837] [Devices]
Hi,

My Nous-A1Z devices are reporting that the installed version is 192. That is nice. Too bad that this is not reported in the OTA screen of Z2M. The value is empty. How to fix this?

Information:
{
    "child_lock": "UNLOCK",
    "current": 0,
    "energy": 0.74,
    "indicator_mode": "off/on",
    "linkquality": 65,
    "power": 0,
    "power_outage_memory": "restore",
    "state": "ON",
    "update": {
        "installed_version": 192,
        "latest_version": 192,
        "state": "idle"
    },
    "voltage": 235
}

Screen:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/6578301/14e582b7-213e-41b0-87f5-ec159c27720c)

**Comment by Veldkornet** [comment]
Could be that the devices aren't reporting everything that's needed to the frontend. 

I think you need to log it there:

https://github.com/nurikk/zigbee2mqtt-frontend/issues
